src/palette/view/widgets/editdrumsetdialog.cpp

```diff
@@ -375,11 +375,14 @@ void EditDrumsetDialog::shortcutChanged()
     }
 
     int pitch = item->data(Column::PITCH, Qt::UserRole).toInt();
+    int index = shortcut->currentIndex();
+    bool invalidIndex = index < 0 || index >= 7;
```

Comment:
I think we can move "ABCDEFG" to a static variable and use it everywhere (size, referencing by index below)

Reply:
done